source-map.bs

```diff
@@ -168,7 +168,7 @@ for practical reasons, the order cannot be defined in many JSON generators and
 has never been enforced.
 
 <dfn><code>version</code></dfn> is the version field which is always the number
-`3` as an integer.
+`3` as an integer. Source map can be rejected in case of a value different from `3`.
```

Comment:
...which *must always be* the number `3` as an integer. The source map *may* be rejected in case of a value different from `3`.